### DISCUSSION TITLE
Add PayPal gateway support

### DISCUSSION THREAD

**Discussion by remcotolsma** [pronamic/wp-pronamic-pay#110] [Ideas]
Mainly interesting for Pronamic to be able to find all payments in the WordPress admin dashboard under "Pay" » "Payments". And to ensure that PayPal payments can also be tracked via the Google Analytics module.

https://github.com/wp-pay/core/blob/develop/src/GoogleAnalyticsEcommerce.php

This is already a work in progress gateway: https://github.com/wp-pay-gateways/paypal.

Please upvote if you are also interested in a direct integration with PayPal.

## Links

- https://basecamp.com/1810084/projects/18117394/todos/441785357
- https://www.wp-pay.org/discussions/add-paypal-gateway-support/

**Comment by rvdsteege** [comment]
Support for PayPal has recently been added in Pronamic Pay version 6.9.0.

- https://github.com/pronamic/wp-pronamic-pay/blob/6.9.0/CHANGELOG.md#690---2021-08-09
